apollo-router/src/uplink/license_enforcement.rs

```diff
@@ -240,14 +281,19 @@ impl LicenseEnforcementReport {
 
     fn schema_restrictions() -> Vec<SchemaRestriction> {
         vec![
-            SchemaRestriction::builder()
-                .name("@authenticated")
-                .url("https://specs.apollo.dev/authenticated/v0.1")
-                .build(),
-            SchemaRestriction::builder()
-                .name("@requiresScopes")
-                .url("https://specs.apollo.dev/requiresScopes/v0.1")
-                .build(),
+            SchemaRestriction::Directive {
+                name: "@authenticated".to_string(),
+                url: "https://specs.apollo.dev/authenticated/v0.1".to_string(),
```

Comment:
a topic for a follow up PR: should the schema restrictions also work on a version range? Or a URL prefix like `https://specs.apollo.dev/authenticated/`?

Reply:
Thanks for pointing this out. I resolved a couple related bugs and implemented the version range in a separate PR.
https://github.com/apollographql/router/pull/4570